src/Validation/Validation.php

```diff
@@ -1098,6 +1098,68 @@ public static function longitude($value, array $options = [])
     }
 
     /**
+     * Check that the input value is within the ascii byte range.
+     *
+     * This method will reject all non-string values.
+     *
+     * @param string $value The value to check
+     * @return bool
+     */
+    public static function ascii($value)
```

Comment:
@dereuromark well I might be wrong but up to now none if rules use the `is` prefix so should rather keep it as is and change `isInteger` to `integer`.